tools/topology/platform/intel/icl.m4

```diff
@@ -47,4 +47,38 @@ define(`SSP_NAME', `SSP0-Codec')
 undefine(`SSP_MCLK_RATE')
 define(`SSP_MCLK_RATE', `19200000')
 
+
+
+#--------- SSP1 --------------
+#SSP setting for CML platform
+define(`SSP1_INDEX', 1)
+define(`SSP1_NAME', `SSP1-Codec')
+define(`SSP1_MCLK_RATE', `24000000')
+
+ifelse(SOF_ABI_VERSION_3_9_OR_GRT, `1',
+    define(`SSP1_VALID_BITS', 24),
+    define(`SSP1_VALID_BITS', 16))
+
+# playback DAI is SSP1 using 2 periods
+# Buffers use s16le format, with 48 frame per 1000us on core 0 with priority 0
+# With m/n divider available we can support 24 bit playback
+ifelse(SOF_ABI_VERSION_3_9_OR_GRT, `1',
+    define(`SSP1_VALID_BITS_STR', s24le),
+    define(`SSP1_VALID_BITS_STR', s16le))
+
+ifelse(SOF_ABI_VERSION_3_9_OR_GRT, `1',
+    define(`SSP1_BCLK', 2304000),
+    define(`SSP1_BCLK', 1500000))
```

Comment:
@fredoh9 is the below not possible ? Note, its using quoted text.
```
ifelse(SOF_ABI_VERSION_3_9_OR_GRT, `1',
    `# New ABI settings
     define(`SSP1_VALID_BITS_STR', s24le)
     define(`SSP1_BCLK', 2304000)',
    `# Old ABI settings
     define(`SSP1_VALID_BITS_STR', s16le)
     define(`SSP1_BCLK', 1500000)')
```

Reply:
@lgirdwood yes, I can put all define in single condition. What will be better idea, separate ABI file or use single(or less ifelse)?

Reply:
Keep a single platform file and use one condition. You can use ```#comment``` to help the flow of your code so that readers can see any deltas quickly. This comment is also seen on the output file too which also helps readability.

Reply:
While Keeping single cml.m4, only one ifelse is used. All necessary defines and comments are put together.